src/components/url-input-popover/index.js

```diff
@@ -9,27 +9,52 @@ import {
 	PanelBody,
 	ToggleControl,
 	TextControl,
+	Button,
 } from '@wordpress/components'
-import { URLInput } from '@wordpress/block-editor'
+import {
+	URLInput, __experimentalLinkControl as LinkControl,
+} from '@wordpress/block-editor'
+import { keyboardReturn } from '@wordpress/icons'
 import { __ } from '@wordpress/i18n'
 
 /**
  * External dependencies
  */
 import { i18n } from 'stackable'
 import classnames from 'classnames'
+import _ from 'lodash'
```

Comment:
Please import the specific functions used instead of the main one.

For example:
```suggestion
import { omit, omitBy, keys } from 'lodash'
```